src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -268,6 +268,17 @@ extern void setDbPrefix(std::string _dbPrefix)
 
 /*****************************************************************************
 *
+* dbPrefixGet -
+*/
+const char* dbPrefixGet(void)
```

Comment:
Sure, about the name change, even though much better would be to change setDbPrefix to dbPrefixSet - object-action, a more 'object-oriented' naming convention that I use since ... since ever.
We'll talk more about this when discussing the style-guide.

Now, NO function should ever return a std::string. This is something we do a lot, but we really need to stop sending **objects** over the stack. Once we have lint or cppcheck we will get a lot of complaints about this.
The user of dbPrefixGet don't need/want std::string and it would just be a complete waste of time (and especially run-time) to make it return a std::string.
If we want them to use the same type (I agree that homogenity is a good thing), I would propose to change setDbPrefix from std::string to char*.

Either that or we try to return a "const std::string&", which I don't fully understand. However, if we look for speed, char* is the way to go. [ not that this function will even slightly modify the total speed of the broker ]